Cargo.toml

```diff
@@ -22,18 +22,14 @@ travis-ci = { repository = "gtk-rs/gio" }
 name = "gio"
 
 [features]
-v2_34 = ["gio-sys/v2_34", "glib/v2_34"]
-v2_36 = ["v2_34", "gio-sys/v2_36", "glib/v2_36"]
-v2_38 = ["v2_36", "gio-sys/v2_38", "glib/v2_38"]
-v2_40 = ["v2_38", "gio-sys/v2_40", "glib/v2_40"]
-v2_42 = ["v2_40", "gio-sys/v2_42"]
-v2_44 = ["v2_42", "gio-sys/v2_44", "glib/v2_44"]
+v2_44 = ["gio-sys/v2_44", "glib/v2_44"]
 v2_46 = ["v2_44", "gio-sys/v2_46", "glib/v2_46"]
 v2_48 = ["v2_46", "gio-sys/v2_48", "glib/v2_48"]
 v2_50 = ["v2_48", "gio-sys/v2_50", "glib/v2_50"]
 v2_52 = ["v2_50", "gio-sys/v2_52", "glib/v2_52"]
 v2_54 = ["v2_52", "gio-sys/v2_54", "glib/v2_54"]
-v2_56 = ["v2_54", "gio-sys/v2_56"]
+v2_56 = ["v2_54", "gio-sys/v2_56", "glib/v2_56"]
+v2_58 = ["v2_56", "gio-sys/v2_58", "glib/v2_58"]
```

Comment:
Might be worth mentioning in the release notes separately that the minimum supported version of each crate is the GNOME 3.14 version (Sep 2014) and we updated the bindings up to version GNOME 3.30.